restli-int-test-api/src/main/snapshot/com.linkedin.restli.examples.greetings.client.typerefCustomDoubleAssociationKeyResource.snapshot.json

```diff
@@ -26,6 +26,11 @@
     "ref" : "double",
     "java" : {
       "class" : "com.linkedin.restli.examples.custom.types.CustomDouble"
+    },
+    "validate" : {
+      "regex" : {
+        "regex" : "[0-9]*\\.[0-9]"
```

Comment:
Question. Why is not not "[0-9]*\\.[0-9]*"   how many digits are allowed after the "."?

Reply:
yup that is only 1